awx/api/tests/functional/test_api_generics.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+# We have to tell flake 8 to ignore these since they are fixtures we need to import
+from awx.conf.tests.functional.conftest import api_request, admin  # noqa: F401; pylint: disable=unused-variable
+
+
+@pytest.mark.django_db
+def test_change_400_error_log(caplog, api_request, settings):
+    settings.API_400_ERROR_LOG_FORMAT = 'Test'
```

Comment:
I think it's best to use `overwrite_settings` like this

https://github.com/ansible/awx/blob/eec4f8dcc27428cd1aeb1c251d43052a3ea780e8/awx/main/tests/functional/commands/test_secret_key_regeneration.py#L34